cfme/tests/containers/test_projects_dashboard.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.container import Container
+from cfme.containers.image import Image
+from cfme.containers.service import Service
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.9"),
```

Comment:
**Required**

Do not use `current_version()`, use appliance fixture's version attribute.

That aside, I think you would be better off using `pytest.mark.ignore_stream('5.8')` here.

Reply:
Use pytest.mark.ignore_stream('5.8') as you suggest.
It definitely looks better.